rules-java/src/main/resources/reports/templates/ignored_files.ftl

```diff
@@ -13,12 +13,14 @@
             <tr>
                 <th>File</th>
                 <th>Path</th>
+		<th>Ignored by regex</th>
             </tr>
 
             <#list reportModel.ignoredFiles.iterator() as file>
             <tr>
                 <td> <#if file.fileName?has_content> ${file.fileName} </#if> </td>
                 <td> <#if file.filePath?has_content> ${file.filePath} </#if> </td>
+		<td> <#if file.ignoredRegex?has_content> ${file.ignoredRegex} </#if> </td>
```

Comment:
Does this also specify the ignore file that the regex came from? Do you think that's a good idea?